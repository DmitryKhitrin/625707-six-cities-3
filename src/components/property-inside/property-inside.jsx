import React from "react";
import PropTypes from "prop-types";

export const ProppertyInside = ({
  property = ``,
}) => {
  return (
    <li className="property__inside-item">{property}</li>
  );
};

ProppertyInside.propTypes = {
  property: PropTypes.string.isRequired
};
