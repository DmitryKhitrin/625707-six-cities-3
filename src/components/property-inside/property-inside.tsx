import React, {FC} from "react";

type Props = {
  property: string;
}

export const ProppertyInside: FC<Props> = ({
  property = ``,
}) => {
  return (
    <li className="property__inside-item">{property}</li>
  );
};
