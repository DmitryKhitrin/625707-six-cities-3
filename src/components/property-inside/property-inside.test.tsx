import React from "react";
import renderer from "react-test-renderer";
import {ProppertyInside} from "./property-inside";

const mocData = `Душик`;

it(`<ProppertyInside /> should render main.`, () => {
  const tree = renderer.create(<ProppertyInside property={mocData} />).toJSON();
  expect(tree).toMatchSnapshot();
});
