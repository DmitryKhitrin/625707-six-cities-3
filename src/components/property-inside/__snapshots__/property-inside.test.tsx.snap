// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProppertyInside /> should render main. 1`] = `
<li
  className="property__inside-item"
>
  Душик
</li>
`;
